test/specs/core.scaleService.tests.js

```diff
@@ -2,14 +2,15 @@
 describe('Test the scale service', function() {
 
 	it('should update scale defaults', function() {
-		var defaults = {
-			testProp: true
-		};
 		var type = 'my_test_type';
 		var Constructor = function() {
```

Comment:
We are not requiring custom scale to extend core.scale, should we?

Reply:
I think it's a reasonable requirement so that we can check what type of object it is in `Chart.register` as you suggested above. I'll wait to change this until we get around to implementing that though